app/src/main/java/ar/com/wolox/android/example/network/requests/UserAdapter.java

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.example.network.requests;
+
+import ar.com.wolox.android.example.network.services.UserService;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * User Adapter
+ */
+public class UserAdapter {
```

Comment:
This is Ok, but you should look for BASE_URL in your code. This is already set, you just need to change it. Let's talk about this together.